### PR TITLE
Uniformisation blank in quizzes and in adventures

### PR DESCRIPTION
**Pull request by thjazi39**
As said in issue #592, a little standardization of syntax files

Uniformity of blanks in the codes:
- In quizzes, there were "\_\_\_" left (whereas we want "\_?\_"
- in adventures, there were "\_\_" or "\_\_\_" (instead of "\_")

### CODE REVIEW

File: coursedata/quizzes/el.yaml [hunk 1]
```diff
@@ -62,16 +62,16 @@ levels:
             question_score: '10'
         10:
             mp_choice_options:
```
Comment (Felienne) [inline coursedata/quizzes/el.yaml:64]: These texts are generated by Weblate. I guess to minimize conflicts, we want to remove these form the PR and just make the change in En and the actually translated quizzes (such as nl, for example)

**Comment by thjazi39** [comment]
Ok, so I have to report the changes to the site that does the translations?
There are only 115 changes to make, I think it will be quick to do by hand

**Comment by Felienne** [comment]
> Ok, so I have to report the changes to the site that does the translations? There are only 115 changes to make, I think it will be quick to do by hand

No that will go automatically but smaller changes are better. So if you can just remove all quizzes that are 100% copies on En from the PR (for example Greek, but I do not know then all by heart), everything will be fine!

**Comment by thjazi39** [comment]
Ok, I think this is better now

**Review by Felienne — APPROVED** [review]
Hope this does not upset Weblate 🤞

**Pull request merged**